core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,796 @@
-
-          <div class="credits oppia-about-tab-content" [ngClass]="{'oppia-about-visible-content': activeTabName === TAB_ID_CREDITS}">
-            <div class="oppia-static-card-content oppia-static-card-content-wide">
-              <h2 [innerHTML]="'I18N_ABOUT_PAGE_CREDITS_TAB_HEADING' | translate"></h2>
-              <p [innerHTML]="'I18N_ABOUT_PAGE_CREDITS_TAB_TEXT' | translate">
-              </p>
-              <div class="oppia-about-credits-letter-groups three-col">
-                <div *ngFor="let credit of allCredits">
-                  <span>{{ credit.letter }}</span>
-                  <ul>
-                    <li *ngFor="let name of credit.names">{{ name }}</li>
-                  </ul>
-                </div>
-              </div>
-              <p [innerHTML]="'I18N_ABOUT_PAGE_CREDITS_TAB_TEXT_BOTTOM' | translate :{listOfNames: listOfNames}">
-              </p>
-              <p [innerHTML]="'I18N_ABOUT_PAGE_CREDITS_THANK_TRANSLATEWIKI' | translate">
-              </p>
-            </div>
+  <div class="oppia-about-page-section-two oppia-about-section">
+    <div class="oppia-about-section-two-top oppia-about-fixed-section">
+      <div class="oppia-section-two-content">
+        <h3 class="oppia-about-title oppia-about-page-title" [innerHTML]="'I18N_ABOUT_PAGE_OUR_FEATURES' | translate"></h3>
+        <p class="oppia-about-content oppia-about-page-content" [innerHTML]="'I18N_ABOUT_PAGE_OUR_FEATURES_CONTENT' | translate"></p>
+      </div>
+      <div class="oppia-about-icons">
+        <div class="oppia-about-features-container oppia-flex oppia-flex">
+          <div class="oppia-about-feature-description-container oppia-flex oppia-flex-col" *ngFor="let feature of features let i = index">
+            <img [src]="getStaticImageUrl(feature.imageFilename)" alt="{{feature.i18nDescription | translate}}" class="oppia-feature-icon">
+            <div class="oppia-about-feature-text" [innerHTML]="feature.i18nDescription | translate"></div>
           </div>
         </div>
       </div>
-
-      <div class="oppia-about-extra-container">
-        <div class="oppia-static-extra-content oppia-about-clearfix">
-          <div class="oppia-about-otter-container">
-            <picture>
-              <source type="image/webp" [srcset]="getStaticImageUrl('/general/about_page_mascot.webp')">
-              <source type="image/png" [srcset]="getStaticImageUrl('/general/about_page_mascot.png')">
-              <img [src]="getStaticImageUrl('/general/about_page_mascot.png')" class="oppia-about-otter" alt="Otter">
-            </picture>
-          </div>
-          <div class="oppia-about-otter-speech-card">
-            <blockquote class="oppia-about-otter-speech-bubble">
-              <h3 [innerHTML]="'I18N_ABOUT_PAGE_SPEECH_BUBBLE' | translate"></h3>
-            </blockquote>
+    </div>
+  </div>
+  <div class="oppia-about-page-section-three oppia-about-section">
+    <div class="oppia-section-three oppia-about-fixed-section">
```

Comment:
Where is this `oppia-section-three` being used? Can you please self-review this PR and ensure it's in a good state? :) This will help you get your PR merged quickly!

Reply:
removed